package.json

```diff
@@ -4,26 +4,24 @@
   "engines": {
     "node": "9.11.1"
   },
-  "dependencies": {},
+  "dependencies": {
+    "View": "git+ssh://git@github.com/FutureDaysSoftware/View.git#bundling"
+  },
   "devDependencies": {
     "babel-cli": "^6.26.0",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-stage-2": "^6.24.1",
-    "babelify": "8.0.0",
-    "browserify": "14.5.0",
-    "uglify-js": "^3.3.21",
-    "uuid": "^3.2.1",
-    "view": "git+ssh://git@github.com/FutureDaysSoftware/View.git#slurper"
+    "stylus": "^0.54.5"
   },
   "license": "private",
-  "main": "dist/bundle.js.gz",
+  "main": "dist/js/Toast.js",
```

Comment:
`dist/js/index.js`

We are already in the `Toast` repo, the main file should be named `index`